source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,20 @@
 ============
 Introduction
 ============
+
+oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
+ from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.
+
+Library consists of the following major components: Data Management, Algorithms, and Services
```

Comment:
makes sense to link to the relevant sections for each component

also: a full stop is missing at the end of a sentence

Reply:
yes, makes sense to have structured view:

The library consists of ..

-  **Data Management** ...
- Algorithms
- 
or similar

Reply:
for now i've made structural view
![image](https://user-images.githubusercontent.com/18015221/77201844-4b8d2480-6afe-11ea-8424-e5c42cf5fe4f.png)

For linking to section - just make it as hyperlink to corresponding document?  Is there place there this is  done already, so i will make it in consistent way?

Reply:
use `.. _label:` to label a particular place in the document
to refer readers there, use one of the following:
```
:ref:`label`
:ref:`alt text <label>`
```
[example](http://intel.github.io/daal/dev_guide/algorithms/lasso_elastic_net/elastic-net.html) (first paragraph)

Reply:
Thanks!. done